nautobot/dcim/models/device_component_templates.py

```diff
@@ -372,6 +374,8 @@ class FrontPortTemplate(ModularComponentTemplateModel):
         ],
     )
 
+    natural_key_field_names = ["name", "device_type", "module_type", "rear_port_template", "rear_port_position"]
```

Comment:
Should this be, for consistency?

```suggestion
    natural_key_field_names = ["device_type", "module_type", "name", "rear_port_template", "rear_port_position"]
```